src/mumble/MainWindow.h

```diff
@@ -431,6 +433,7 @@ public slots:
 	void openAboutDialog();
 	void openAboutQtDialog();
 	void versionCheck();
+	void togglePositionalAudio(const bool &newState);
```

Comment:
Why do you take the boolean parameter by reference? This seems unnecessary as the size of a boolean is quite a bit smaller than the size of a pointer (which is what a reference really is, under the hood) and therefore passing the boolean by value should be more efficient.
Premature optimizations aside, I think it would also be clearer to pass by value as passing a boolean by reference introduces the semantics that it shall act as some kind of out-parameter (though that is of course not possible here due to the `const` qualification).

So in short: Unless you have good reason to do so, I would prefer if we passed the boolean by value instead of by reference.